Gemfile

```diff
@@ -26,6 +29,7 @@ gem 'capistrano-db-tasks', {
 group :development, :test do
   gem 'pry-rails'
   gem 'rspec-rails', '~> 4.0.0'
+  gem "byebug", "~> 11.1"
```

Comment:
this is superfluous next to `pry-rails`, which lets you drop `binding.pry` in code.